toto/runlib.py

```diff
@@ -100,6 +100,11 @@ def record_artifacts_as_dict(artifacts):
     return artifacts_dict
 
   for artifact in artifacts:
+
+    if not os.path.exists(artifact):
+      log.warning("path: {} does not exist, skipping..".format(artifact))
+      continue 
+
```

Comment:
I don't mean os.path.lexists, which is pretty much the same in many cases.

I don't know if we want to record them as artifacts, but I guess we would want to have a behavior defined for it.

`isfile` does follow links, but that's just one case of special file.